### DISCUSSION TITLE
Scan interval configuration question

### DISCUSSION THREAD

**Discussion by jonteriele** [Hyundai-Kia-Connect/kia_uvo#804] [General]
Hi there,

Is there a specific reason to have a minimum of 15 minutes for the _Scan Interval_ setting?
It would be nice to have more up-to-date location information.

Thanks in advance.

**Comment by naThnaabu** [comment]
I am wondering in general how this works.
How often is Bluelink itself updating?

If I park my car in an underground garage I expect it to have a location at least near that parking and not the city 3 states over.

**Comment by cdnninja** [comment]
Few reasons to not scan too often. 

1. Force scans consume your cars 12v and can kill it. 
2. API calls are limited per day by the server.

**Reply by jonteriele** [reply]
Hi, 

Thanks for the reply.

1. Does this also count for regular scans? It would be nice f.e. to scan more often only when the car is driving,
2. Ah too bad. This makes it impossible to have an accurate position of the car so that I can build some nice automations based on the location.

**Reply by naThnaabu** [reply]
I don't understand how tesla is fine with live location and Hyundai cars are not

**Reply by cdnninja** [reply]
Yes all api calls count.

**Comment by TheHuskyCrobats** [comment]
There is any way to force update all information?
For example, my car was at 25km from home and charging for 1 hour and in HA it was showing at home and unplugged.
